src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateGoalsTest.kt

```diff
@@ -178,7 +178,7 @@ class CreateGoalsTest : IntegrationTestBase() {
       "notesCharacterCount" to "23",
     )
     await.untilAsserted {
-      verify(telemetryClient).trackEvent("goal-create", expectedEventCustomDimensions, null)
+      verify(telemetryClient).trackEvent("goal-created", expectedEventCustomDimensions, null)
```

Comment:
I've changed the existing App Insight events into past tense verbs (ie: `goal-create` -> `goal-created`, and `goal-update` -> `goal-updated`)

This felt more comfortable / better naming when it came to naming the event for step removed. If I'd have keep with the current naming it would have been `step-remove` which feels weird

Given we are not live yet I felt it was safe/OK to rename events (ie will be a different story once we are live, so getting this right now feels important)